helm-org.el

```diff
@@ -282,27 +282,36 @@ current heading."
 ;; if its name starts with "helm"?!  If its name starts with anything
 ;; else, it works fine!
 (defun not-helm-org-completing-read-tags (prompt coll pred req initial hist def inh)
-  (let* ((initial (and (stringp initial)
-                       (not (string= initial ""))
-                       initial))
-         (curr (when initial
-                 (org-split-string initial ":")))
-         (table (org-uniquify (mapcar 'car org-last-tags-completion-table)))
-         (table (if curr
-                    ;; Remove current tags from list
-                    (cl-delete-if (lambda (x)
-                                    (member x curr))
-                                  table)
-                  table))
-         (prompt (if initial
-                     (concat "Tags " initial)
-                   prompt)))
-    (concat initial
-            (mapconcat 'identity
-                       (nreverse (helm-org-completing-read-multiple
-                                  prompt table pred nil nil hist def
-                                  t "Org tags" "*Helm org tags*" ":"))
-                       ":"))))
+  (if (not (string= "Tags: " prompt))
```

Comment:
> Hmm, I have no prompt in org-capture, perhaps because I am using org-capture-templates ?

Depending on the template, yes.  For example, this template prompts for a heading ("Description"):

```el
("i" "Inbox: general entry" entry
  (file "")
  "* %^{Description} %^G\n%U %?")
```